target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -259,6 +263,10 @@ case "$FIRMWARE" in
 		caldata_extract "ART" 0x5000 0x2f20
 		ath10k_patch_mac $(mtd_get_mac_binary dnidata 0xc)
 		;;
+	netgear,wac510)
+		caldata_extract "0:ART" 0x5000 0x2f20
+		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary "0:MANUDATA" 0x6) +0x10)
```

Comment:
Just "16" instead of "+0x10" would be much nicer.
Is the proper address really not present on the flash anywhere?

Reply:
You probably meant +16 but, yeah, I get your point. Thanks!

Reply:
I meant "16", no reason to add a "+".

What about the address on flash.